Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/lawbringer.yml

```
@@ -17,18 +23,13 @@
     layers:
     - state: omnilaser-yellow
       shader: unshaded
-  - type: Projectile
-    damage:
-      types:
-        Blunt: 25
```

Blunt is easier to heal than pierce.
```suggestion
  - type: Projectile
    damage:
      types:
        Blunt: 14